playground/helpers/utils.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+import json2csv from 'json2csv';
+import moment from 'moment';
+
+const writeFileAsync = promisify(fs.writeFile);
+const makeDirAsync = promisify(fs.mkdir);
+const accessAsync = promisify(fs.access);
+
+export function promisify(fn) {
```

Comment:
@esakal this function already exists under `util` module, see [here](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original)